src/api/subscriptions.js

```diff
@@ -257,6 +257,27 @@ export const WindowOpenMode = {
   REDIRECT: 'redirect',
 };
 
+/**
+ * @enum {string}
+ */
+export const ReplaceSkuProrationMode = {
+  // The replacement takes effect immediately, and the remaining time will
+  // be prorated and credited to the user. This is the current default
+  // behavior.
+  IMMEDIATE_WITH_TIME_PRORATION: 'IMMEDIATE_WITH_TIME_PRORATION',
+};
+
+/**
+ * String values input by the publisher are mapped to the number values.
+ * @enum {number}
+ */
+export const ReplaceSkuProrationModeMapping = {
```

Comment:
This is an internal mapping detail, so we should move closer to where it's used within the runtime.

Reply:
done